cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -1,16 +1,19 @@
 """
 Shard analysis and rebalancing logic for CrateDB
 """
-
+import datetime
```

Ruff (0.12.2)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)